apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistSplitter.tsx

```diff
@@ -0,0 +1,5 @@
+import tokens from '@contentful/f36-tokens';
+
+export default function ChecklistSplitter() {
+  return <div style={{ borderBottom: `1px solid ${tokens.gray300}` }} />;
```

Comment:
nit: seems like we are following a pattern of having a separate styles file instead of inline styling. But not imperative to adjust.

Reply:
done